### PR TITLE
Media stack service

### PR DESCRIPTION
**Pull request by Miszo97**
Add class MediaStackService that encapsulates all http calls to a news provider.

I used VCR to test the external API.

### CODE REVIEW

File: backend/services/media_stack_service.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+        if source:
+            url += f"&sources={source.value}"
+
+        if limit:
+            url += f"&limit={limit}"
+
+        if offset:
+            url += f"&limit={offset}"
+
```
Comment (Miszo97) [inline backend/services/media_stack_service.py:41]: Is there a better way of creating url from parameters than this approach?

File: backend/tests/services/cassettes/test_get_articles_by_parameters.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      User-Agent:
+      - python-requests/2.27.1
+    method: GET
+    uri: http://api.mediastack.com/v1/news?access_key=162b82c434e7ba8482f5b81b8d3d6334&countries=us&languages=en&categories=general&sources=cnn&limit=25
```
Comment (tomkeee) [inline backend/tests/services/cassettes/test_get_articles_by_parameters.yaml:14]: We have to find a solution to make our news_endpoint and api_key secure

File: backend/tests/services/test_media_stack_service.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+def test_get_articles_limit():
+    articles = media_stack_service.get_news_by_parameters(
+        country=Country.US,
+        language=Language.EN,
+        category=Category.GENERAL,
+        source=Source.CNN,
+        limit=5,
+    )
+
+    assert len(articles) == 5
```
[the remark this answers is not in the document]
Comment (tomkeee) [reply backend/tests/services/test_media_stack_service.py:36]: Maybe we should add tests with missing/invalid data, and assert the response status

**Review by tomkeee — CHANGES_REQUESTED** [review]

File: backend/Pipfile [hunk 4]
```diff
@@ -17,6 +17,9 @@ passlib = "*"
 python-jose = "*"
 python-multipart = "*"
 sqlalchemy_utils = "*"
+vcrpy = "*"
+pip = "*"
+pytest-vcr = "*"
```
[the remark this answers is not in the document]
Comment (Miszo97) [reply backend/Pipfile:22]: You are right. Thanks!

File: backend/tests/services/test_media_stack_service.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (Miszo97) [reply backend/tests/services/test_media_stack_service.py:36]: Good point!

[the remark this answers is not in the document]
Comment (Miszo97) [reply backend/tests/services/test_media_stack_service.py:36]: Wait. What status? We don't test endpoints here :D

[the remark this answers is not in the document]
Comment (tomkeee) [reply backend/tests/services/test_media_stack_service.py:36]: Yeah, that's true, my bad 😅

**Review by tomkeee — APPROVED** [review]

**Comment by tomkeee** [comment]
Well Done 💪  !

**Pull request merged**